taxonomy-editor-frontend/src/pages/allentries/EntriesList.jsx

```diff
@@ -0,0 +1,84 @@
+import { createTheme, ThemeProvider, Typography } from "@mui/material";
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import Paper from '@mui/material/Paper';
+import EditIcon from '@mui/icons-material/Edit';
+import IconButton from '@mui/material/IconButton';
+import { useNavigate } from "react-router-dom";
+
+const EntriesList = ({ nodes, title }) => {
+    const navigate = useNavigate();
+    const handleClick = (event, id) => {
+        event.preventDefault();
+        navigate('/entry/'+id);
+    }
+    const theme = createTheme({
+        typography: {
+            fontFamily : 'Plus Jakarta Sans',
```

Comment:
maybe use this at the top of the tree? ... like in the App.jsx ?, that way you get a general style for all the app